README.md

```diff
@@ -26,15 +26,15 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 ```
 
 ```text
-Your answer goes here...
+They do the same thing but most HTML validators will require the "data" prefix to pass for validation.
 ```
 
 ### Question 3
 
 Which of the three following options demonstrates the best usage of `ng-app`? **Explain your answer.**
 
 ```text
-Your answer goes here...
+A because the ng-app always goes in the HTML tag to let the document know it is going to be an ng-app with the name of "***" so it can read it from the js files as well.
 ```
 
```

Comment:
The div is the better choice since multiple angular apps could be on a page simultaneously